source/adapters/level_zero/v2/latency_tracker.hpp

```diff
@@ -0,0 +1,212 @@
+//===--------- ur_latency_tracker.cpp - common ---------------------------===//
+//
+// Copyright (C) 2024 Intel Corporation
+//
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM
+// Exceptions. See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <chrono>
+#include <cstdint>
+#include <limits>
+#include <memory>
+
+#include "logger/ur_logger.hpp"
+
+#if defined(UR_ENABLE_LATENCY_HISTOGRAM)
+
+#include <hdr/hdr_histogram.h>
+
+namespace v2 {
+
+static inline bool trackLatency = []() {
```

Comment:
since we have the logger, can we control whether we actually collect based on whether the logger verbosity is set high enough (info)?
One less env variable to set :)

Reply:
Good idea, done.